.nais/vars-q0.yaml

```diff
@@ -7,3 +7,6 @@ ingresses:
   - "https://meldekortservice-q0.dev.intern.nav.no"
   - "https://app-q0.adeo.no/meldekortservice"
   - "https://meldekortservice-q0.nais.preprod.local"
+  - "https://meldekortservice-q0.dev-fss-pub.nais.io"
```

Comment:
Må åpnes for sonekryssing hvis vi kan ha flere dev-miljø i GCP

Reply:
Done